integration-tests/src/test/java/tachyon/shell/command/PinCommandTest.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.shell.command;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import tachyon.Constants;
+
+import tachyon.client.TachyonFSTestUtils;
+import tachyon.client.TachyonStorageType;
+import tachyon.client.UnderStorageType;
+import tachyon.client.file.TachyonFile;
+
+import tachyon.shell.AbstractTfsShellTest;
+
+import tachyon.TachyonURI;
+
+public class PinCommandTest extends AbstractTfsShellTest {
+  @Test
```

Comment:
could you also javadoc briefly this testcase?
Exceptions are not required to comment for integration tests.